cmd/crictl/config.go

```diff
@@ -66,37 +71,46 @@ var configCommand = &cli.Command{
 				logrus.Fatalf("No section named %s", get)
 			}
 			return nil
-		}
-		key := context.Args().First()
-		if key == "" {
-			return cli.ShowSubcommandHelp(context)
-		}
-		value := context.Args().Get(1)
-		switch key {
-		case "runtime-endpoint":
-			config.RuntimeEndpoint = value
-		case "image-endpoint":
-			config.ImageEndpoint = value
-		case "timeout":
-			n, err := strconv.Atoi(value)
-			if err != nil {
-				logrus.Fatal(err)
+		} else if context.IsSet("set") {
```

Comment:
I like the new --set sub command for config and the ability to set multiple end points via separator.

Suggestion:  make the --set sub command also work by default... so command (--set optional)... and add the separator option to the get sub command.

Reply:
Good point. Re-added the default set behaviour.